ui/src/app/cron-workflows/cron-workflow-list.tsx

```diff
@@ -180,10 +181,16 @@ export function CronWorkflowList({match, location, history}: RouteComponentProps
                                             )}
                                         </div>
                                         <div className='columns small-1'>
-                                            <Timestamp date={w.metadata.creationTimestamp} />
+                                            <Timestamp date={w.metadata.creationTimestamp} timestampKey={TIMESTAMP_KEYS.CRON_WORKFLOW_LIST_CREATION} />
```

Comment:
The `time format switcher` button here is not part of the header like on all other list pages

![image](https://github.com/user-attachments/assets/a4594ee5-7a5f-4b98-ba53-626116a35912)